src/components/TextField/TextField.tsx

```diff
@@ -47,7 +48,7 @@ export const TextField: React.FC<Props> = ({
           })}
           placeholder={placeholder}
           value={value}
-          onChange={event => onChange(event.target.value)}
+          onChange={event => onChange(event)}
```

Comment:
Will works same but less callbacks
```suggestion
          onChange={onChange}
```